test/generator/filler/test_player_state.py

```diff
@@ -35,13 +35,16 @@ def state_for_blank(
 def test_current_state_report(state_for_blank):
     result = state_for_blank.current_state_report()
     assert result == (
-        "At Intro/Starting Area/Event - Post Weapon after 0 actions and 0 pickups, "
-        "with 4 collected locations, 24 safe nodes.\n\n"
```

Comment:
Would be nice to have the count of safe uncollected resource nodes I guess?

Reply:
good call